spec/features/permissions/edit_permissions_with_invalid_revision_id_spec.rb

```diff
@@ -0,0 +1,60 @@
+# Tests for editing collection permission with invalid revision ID (failed to get one)
+# and conflicting revision ID (users trying to update the same revision)
+describe 'Edit collection permissions', reset_provider: true, js: true do
+  before :all do
+    collection_permission_to_upload = {
+      group_permissions: [ {
+                             permissions: [ 'read', 'order' ],
```

Comment:
the indentation here seems off, can you make it look more like the other tests or the mocked response?